src/org/omegat/core/matching/NearString.java

```diff
@@ -49,37 +49,81 @@
  */
 public class NearString {
     public enum MATCH_SOURCE {
-        MEMORY, TM, FILES
-    };
+        /** From project memory */
+        MEMORY,
+        /** From external TM in project tm/ folder */
+        TM,
+        /** From files */
+        FILES,
+        /** From sub-segmented match */
+        TM_SUBSEG
```

Comment:
Why this prefix TM_ ?
Subsegments can come from any other source, including project memory, tmx files or external files.  They can even be combination of segments which do not come from same source.

Reply:
> Why this prefix TM_ ? Subsegments can come from any other source, including project memory, tmx files or external files. They can even be combination of segments which do not come from same source.

What is your proposal? `SUBSEG` only?

Reply:
Yes, or SUBSEGMENTS as I did in PR #1224 ; in any case no reason why to keep this prefix which gives a wrong info about where data come from.